### PR TITLE
Support for Alfa bank

### PR DESCRIPTION
**Pull request by zeldigas**
Hi I added support for AlfaBank csv format. Tests are included.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -81,6 +88,24 @@ currency
         Cyrrency
         (if not set, will be extracted from the statement)
 
+alfabank
+--------
+
+bank
+        Bank id
+        (default is 'SberBank')
```
Comment (gerasiov) [inline README.rst]: AlfaBank

File: setup.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
       description=("Russian banks plugins for ofxstatement"),
       long_description=long_description,
       license="GPLv3",
-      keywords=["ofxstatement", "russian", "avangard", "tinkoff", "sberbank"],
+      keywords=["ofxstatement", "russian", "avangard", "tinkoff", "sberbank", 'alfabank'],
```
Comment (gerasiov) [inline setup.py]: Please follow coding style. Do not mix different type of quote-sign.

File: src/ofxstatement/plugins/alfabank.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
+#    AlfaBank (https://www.alfabank.ru) plugin for ofxstatement
+#
+#    Copyright 2017 Dmitry Pavlov <zeldigas@gmail.com>
+#
```
Comment (gerasiov) [inline src/ofxstatement/plugins/alfabank.py]: Didn't you forget to mention me in this copyright note? =)

**Review by gerasiov — CHANGES_REQUESTED** [review]
Please fix thees minor issues.

File: setup.py (hunk 2 above)
Comment (zeldigas) [reply setup.py]: OK, will fix

File: src/ofxstatement/plugins/alfabank.py (hunk 3 above)
Comment (zeldigas) [reply src/ofxstatement/plugins/alfabank.py]: I thought that only author is listed and I don't see contributions  ;).  if you think that plugin structure is a good reason to add you here (of course I took it from tinkoff one to follow single coding style) I'm fine to add you

Comment (gerasiov) [reply src/ofxstatement/plugins/alfabank.py]: I just see, that this code is based on one I wrote. =) Technically this is derived work and you should mention original author. (Not critical for me, but I believe there is some kind of good behavior pattern open source developer should follow.)

**Comment by zeldigas** [comment]
CR fixes pushed

**Review by gerasiov — APPROVED** [review]

**Pull request merged**